code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -598,7 +598,7 @@
 	. = 1
 
 /datum/reagent/medicine/sal_acid/overdose_process(mob/living/M)
-	M.adjustFireLoss(-3*REM, 0)
+	M.adjustBruteLoss(-3*REM, 0)
```

Comment:
shouldn't it actually deal brute on overdose instead of healing it since the description also says `Overdose will worsen existing bruising`?

Reply:
`Overdose will worsen existing bruising` source?

Reply:
```
/datum/reagent/medicine/sal_acid
	name = "Salicyclic Acid"
	description = "Stimulates the healing of severe bruises. Extremely rapidly heals severe bruising and slowly heals minor ones. Overdose will worsen existing bruising."
```

Reply:
Huh, you're right... However, Oxandrolone which is supposed to also worsen existing burns instead gets its burn healing doubled. And the wiki mentions that both Oxandrolone and Saliclyc Acid double their healing instead of dealing more damage.

Reply:
Okay then. Should probably change both their descriptions then, since Oxandrolone ones also says it worsens it.

Reply:
Now I'm confused. Checked tg's wiki and code, both Saliclyc Acid and Oxandrolone have them actually deal damage instead of healing more when overdosing. What should it be then?

Reply:
I mean we did have a rebalance pr a long while ago which changed the code for it. #2995
It doesn't mention the double healing for it though, so i guess it could have been an oversight.
The thing it does mention about overdose is `balance: The overdose of most chems now causes organ damage instead of just reversing the effects of what the chem healed.`

I guess would probably be best to wait for a maintainers decision on that.

Reply:
Yeah, seems like waiting for the higherups to make a decision would be the best option.

Reply:
Healing getting better on OD is intentional. The liver damage is enough of a downside for that.

Reply:
Guess I'll just update the descriptions then.